DS/assignmentds6/Employees.java

```diff
@@ -0,0 +1,42 @@
+package assignmentds6;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Employees {
+    private List<Employee> employeesList = new ArrayList<Employee>();
+
+    // will return the employee based on employeeid
+    public Employee getEmployee(int empId) {
+	return employeesList.get(empId);
+    }
+
+    // will prevent if the employee with same employeeid has been added
+    public void addEmployee(Employee employee) {
+	if (!employeesList.contains(employee))
+	    employeesList.add(employee);
+    }
+
+    public List<Employee> getEmployees() {
+	return employeesList;
+    }
+
+    // will print data of all employees
+    public void printDataOfEmployees() {
+	for (Employee employee : employeesList) {
+	    System.out.println(employee.toString());
```

Comment:
by default when we print the reference it call the toString() method, so no need to explicitly call the toString() method.